### PR TITLE
Docs for adding LayerData tuple to viewer

### PR DESCRIPTION
**Pull request by dstansby**
# Description
This adds some short docs for adding a LayerData tuple to the `napari` viewer.

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change
- [ ] example: I check if my changes works with both PySide and PyQt backends
      as there are small differences between the two Qt bindings.  

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

File: docs/plugins/_layer_data_guide.md [hunk 1]
```diff
@@ -117,4 +117,14 @@ Out[7]:
...
+```
+
+### Adding to the viewer
+
+To add a `LayerData` tuple to the napari viewer, use :meth:`Layer.create`:
+
+```python
+>>> image_layer_data = (data, {'name': 'My Image', 'colormap': 'red'}, 'image')
+>>> viewer = napari.current_viewer()
+>>> viewer.add_layer(Layer(*image_layer_data))
```
Comment (Czaki) [inline docs/plugins/_layer_data_guide.md]:
```suggestion
>>> viewer.add_layer(napari.layers.Layer.create(*image_layer_data))
```

**Review by Czaki — CHANGES_REQUESTED** [review]
small fix to example

**Review by Czaki — APPROVED** [review]

**Review by tlambert03 — COMMENTED** [review]
This is definitely useful to document and be aware of, but there's an important caveat (and the main reason why we don't just use `Layer.create` directly in our internal code in the viewer model):

```python
data = np.random.random((4,10,10))
data_tuple = (data, {'channel_axis': 0})
Layer.create(*data_tuple)
```
gives
```pytb
TypeError: _ImageBase.__init__() got an unexpected keyword argument 'channel_axis'
```

this is why in the PR where I introduced `Layer.create` I [proposed that we drop the channel_axis argument](https://github.com/napari/napari/pull/2556#discussion_r659247491)... since it's the _only_ case in which calling `viewer.add_` actually creates and adds multiple layers.  So, I like adding this to the docs, but this one exception probably needs to be mentioned (since `channel_axis` would technically/currently still be a valid LayerDataTuple

**Comment by kcpevey** [comment]
@dstansby The documentation now lives in a separate repo (https://github.com/napari/docs). Would you be open to moving this PR over there and adding the note about the exception?

**Comment by psobolewskiPhD** [comment]
FYI, there's two posts that outline a way of doing this: https://github.com/napari/napari/pull/5240#issuecomment-1282076004
https://github.com/napari/docs/pull/74

**Comment by dstansby** [comment]
Updated in https://github.com/napari/docs/pull/152

**Pull request closed without merging**